src/services/localSSHService.ts

```diff
@@ -53,18 +60,25 @@ export class LocalSSHService extends Disposable implements ILocalSSHService {
     }
 
     private async initialize() {
+        let failureCode: FailedToInitializeCode | undefined;
         try {
             const locations = await this.copyProxyScript();
             await this.configureSettings(locations);
             this.isSupportLocalSSH = true;
         } catch (e) {
-            this.logService.error(e, 'failed to copy local ssh client.js');
-            if (e.message) {
-                e.message = `Failed to copy local ssh client.js: ${e.message}`;
+            this.logService.error(e, 'failed to initialize');
+            failureCode = 'Unknown';
+            if (e?.code) {
+                failureCode = e.code;
+            }
+            if (e?.message) {
+                e.message = `Failed to initialize: ${e.message}`;
             }
             this.telemetryService.sendTelemetryException(this.hostService.gitpodHost, e);
             this.isSupportLocalSSH = false;
         }
+        const flowData = this.flow ? this.flow : { gitpodHost: this.hostService.gitpodHost, userId: this.sessionService.safeGetUserId() };
+        this.telemetryService.sendUserFlowStatus(this.isSupportLocalSSH ? 'success' : 'failure', { ...flowData, flow: 'local_ssh_config', failureCode });
```

Comment:
Could we add info about local app somehow? i.e. whether it is configured, we though that some exceptions are only triggered in case with local app?